src/Features/CSharp/Portable/CodeRefactorings/UseRecursivePatterns/UseRecursivePatternsCodeRefactoringProvider.cs

```diff
@@ -422,6 +439,29 @@ private static RecursivePatternSyntax RecursivePattern(SubpatternSyntax subpatte
                 commonReceiver = GetInnermostReceiver(left, lastName, static (identifierName, lastName) => identifierName != lastName);
             }
 
+            var iSymbol = model.GetSymbolInfo(leftNames[^1]).Symbol;
```

Comment:
```suggestion
            var symbol = model.GetSymbolInfo(leftNames[^1]).Symbol;
```

Reply:
this also needs doc'ing as well.  i have no idea why you're calling this on the leftmost name.

Reply:
I removed this. I didn't see there was a convenient method that did what I wanted and thought I needed to compare the symbols directly